### PR TITLE
Add blog post on making Pipy as Kubernetes infra container

### PR DESCRIPTION
**Pull request by addozhang**
The infra container known as pause conainer exists in every pod in Kubernetes.  

Our exploration is on making Pipy as this infra container, which acts as an invisible *sidecar*.  In this post, we take this container to register non-java application to Spring Clouer Eureka registry. Besides service registration, it can do more job.

### CODE REVIEW

File: content/en/blog/_posts/2022-04-07-pipy-as-k8s-infra-container-for-service-registration/index.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+Kubelet first creates the infra container, and then other containers in Pod are added to the infra container's network namespace in the form of join namespace to achieve network communication between containers. Below is a depiction of a Pod in a service mesh environment:
+
+<img alt="Image of a pod infra container" src="kubernetes-pod-infra-arch.jpg">
+
+Kubernetes maintains a multi-architecture image that includes support for Windows. For Kubernetes v1.23 the recommended pause image is `k8s.gcr.io/pause:3.6`, which is very small and goes into a pause state once started.
+
+## Pipy as Infrastructure container
+
+With above information and details, we can see that in fact, infrastructure containers are invisible **sidecar**. Our exploration on `Pipy as Infra` series is two fold, one is to implement more infrastructure related functionality on this ubiquitous sidecar to make it a real infrastructure container, another one is to reduce the complexity and resource consumption associated with multiple sidecar containers.
```
Comment (sftim) [inline content/en/blog/_posts/2022-04-07-pipy-as-k8s-infra-container-for-service-registration/index.md:20]: What's Pipy? At this point in the article, I don't know and I think most readers wouldn't. The other question readers will have is along the lines of “&hellip;and why should I care?” - it'd be good to answer both.

**Review by sftim — COMMENTED** [review]
I get the impression that this is about abusing the infra (`pause`) container to perform the same outcome as sidecar injection, but in a more fragile way. Is that broadly right?

I feel that this would work better on the Flomesh blog than for the Kubernetes blog.

**Comment by sftim** [comment]
/hold

I have strong concerns about the suitability of this article for the Kubernetes blog.

**Comment by reylejano** [comment]
Hi @addozhang ,

I'm closing this PR as I agree with @sftim that this blog post is better suited on https://blog.flomesh.io/ 

/close

**Pull request closed without merging**